### DISCUSSION TITLE
School usage

### DISCUSSION THREAD

**Discussion by ghost** [FrostCo/AdvancedProfanityFilter#439] [Q&A] [answered]
Looking to use APF on Chromebooks in a school setting. Is there any way to stop APF's config from automatically opening when our pupils log in? Would ideally like the option to lock certain settings too so that they cannot be disabled.

**Comment by richardfrost** [comment]
I'd love to help accommodate your needs with this, because it would be cool to have it used in a school setting. Are you using anything to automate the install/setup, or will you just put putting it on those Chromebooks manually?

**Reply by ghost** [reply]
Thanks for getting back to me. I'm deploying it through the Google Workspace Admin console.

**Reply by richardfrost** [reply]
You're welcome! I have heard that the Admin console does have support for more advanced management, but I can't seem to find good documentation on it. I also don't have access to Google Workspace Admin, so I might need some help from you with knowing what capabilities it has. If you have links to documentation let me know.

What I'm thinking is that if we can push a configuration on install we could definitely stop it from opening automatically, and we could lock the Options page down with a password (not very secure, but enough to deter most people). If you have more specific needs, I'm open to look at adding some additional granularity to the settings.

**Reply by richardfrost** [reply]
If this is something you want to work on a bit more @markfaulkner1, just let me know.

**Reply by richardfrost** [reply, marked as answer]
@markfaulkner1 Sorry for bringing up such an old topic again. Just in case you, or anyone else who stumbles onto this discussion are interested in this, this feature has now been added. See more details here: https://github.com/FrostCo/AdvancedProfanityFilter/wiki/Deployment-&-Managed-Config

It's still a pretty new feature, so feel free to share any feedback if you do use it. Thanks!
